tests/Doctrine/ODM/MongoDB/Tests/Mapping/Driver/XmlDriverTest.php

```diff
@@ -87,6 +89,29 @@ public function testInvalidPartialFilterExpressions()
             ],
         ], $classMetadata->getIndexes());
     }
+
+    public function testDriver()
+    {
+        $classMetadata = new ClassMetadata('TestDocuments\AlsoLoadDocument');
```

Comment:
Please use the `::class` constants instead f string names (needs a `use` statement on top of the file):

```suggestion
        $classMetadata = new ClassMetadata(AlsoLoadDocument::class);
```